### DISCUSSION TITLE
Malay translation

### DISCUSSION THREAD

**Discussion by mujahidfa** [vuejs-translations/guidelines#53] [General]
I'm starting translation efforts for Malay/Bahasa Melayu.

Repo: https://github.com/mujahidfa/docs-ms

We have a big community of Vue developers in Malaysia but no Malay translations for the docs. Malay docs could also benefit speakers in Brunei/Indonesia/Singapore.

## How to contribute
Malay speakers interested to contribute please feel free to indicate your interest down below (say hi 👋).

You can also send a pull request to the [repo](https://github.com/mujahidfa/docs-ms).

Please read the [panduan terjemahan](https://github.com/mujahidfa/docs-ms/issues/1) before you start.

Feel free to reach out directly to me if that's desirable. Best way to contact me is via [Twitter](https://twitter.com/mujahid_fa).

**Comment by k11q** [comment]
Im in.

**Reply by k11q** [reply]
Would you allow me to write issues? Would open them for each page/files to track work as per workflow recommendations. https://github.com/vuejs-translations/guidelines/blob/main/README.md#workflow-recommendations . Maybe we can start discussing conventions aswell

**Reply by mujahidfa** [reply]
@khairulhaaziq yes, feel free! On conventions, I’ve started a list, but I think it’ll be better to discuss it in an issue.

**Reply by mujahidfa** [reply]
I created an issue to keep track of our translation conventions: [[Baca ini] Panduan terjemahan istilah dalam Bahasa Melayu](https://github.com/mujahidfa/docs-ms/issues/1).

**Comment by harithzainudin** [comment]
im in

**Reply by mujahidfa** [reply]
Terima kasih! Pilih mana-mana halaman yang anda ingin terjemah di [issue ini](https://github.com/mujahidfa/docs-ms/issues/4) dengan meletakkan komen di issue tersebut. Ada dialu-alukan untuk buat pull request setelah siap terjemah!
